internal/db/delegation.go

```diff
@@ -17,9 +17,18 @@ type UpdateOption func(*updateOptions)
 
 // updateOptions holds all possible optional parameters
 type updateOptions struct {
-	subState  *types.DelegationSubState
-	bbnHeight *int64
-	btcHeight *int64
+	subState                *types.DelegationSubState
+	bbnHeight               *int64
+	btcHeight               *uint32
```

Comment:
can you elaborate on why type has changed to `uint32` ?

Reply:
and one follow up question: so for BTC we assume height types to be uint32, but for BBN it's int64 correct?